Issue1〜3の修正

masukazuno opened the pull request:
Issue1〜3の解消

masukazuno committed:
Issue1

masukazuno committed:
Issue2

Issue2

masukazuno committed:
Issue3

Issue3

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -1,14 +1,14 @@
   def index
     getWeek
-    @plan = Plan.new
+    @plan = plan.new
```
```
    @plan = plan.new
```
上記記述は修正の必要はございません。
元の記述に戻しておきましょう。

maimai0401 replied:
修正が確認できませんでした。
再度修正しましょう。

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -1,14 +1,14 @@
   # 予定の保存
   def create
-    Plan.create(plan_params)
+    plan.create(plan_params)
```
```
    plan.create(plan_params)
```
上記記述は修正の必要はございません。
元の記述に戻しておきましょう。

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -22,19 +22,19 @@ def getWeek
     @week_days = []
 
-    plans = Plan.where(date: @todays_date..@todays_date + 6)
+    plans = plan.where(date: @todays_date..@todays_date + 6)
```
```
    plans = plan.where(date: @todays_date..@todays_date + 6)
```
上記記述は修正の必要はございません。
元の記述に戻しておきましょう。

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -22,19 +22,19 @@ def getWeek
 
     # Dateオブジェクトは、日付を保持しています。下記のように`.today.day`とすると、今日の日付を取得できます。
-    @todays_date = Date.today
+    @todays_date = date.today
```
```
    @todays_date = date.today
```
上記記述は修正の必要はございません。
元の記述に戻しておきましょう。

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -22,19 +22,19 @@ def getWeek
         today_plans.push(plan.plan) if plan.date == @todays_date + x
       end
-      days = { :month => (@todays_date + x).month, :date => (@todays_date+x).day, :plans => today_plans}
+      days = { :month : (@todays_date + x).month, :date : (@todays_date+x).day, :plans : today_plans}
```
```
      days = { :month : (@todays_date + x).month, :date : (@todays_date+x).day, :plans : today_plans}
```
正しいシンボル型になっていないようです。
再度修正しましょう。

maimai0401 replied:
https://github.com/masukazuno/training_curriculum_rails7/pull/7/files#r1441515661

修正が確認できませんでした。
修正しましょう。

maimai0401 commented on app/views/calendars/index.html.erb:
```diff
@@ -23,5 +23,4 @@
         </ul>
       </div>
     <% end %>
```
まだファイル内に不要なdivタグが残っているようです。
修正しましょう。

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -1,14 +1,14 @@
-    @plan = Plan.new
+    @plan = plan.new
   end
 
```
issue2の修正が確認できませんでした。
再度修正しましょう。

maimai0401 reviewed (commented):
レビュー依頼ありがとうございます。プルリクエスト確認させていただきました。

修正点をコメントさせて頂きましたので、ご確認をお願い致します。

【次回プルリクエスト提出する際の注意点】
```ruby
①修正が終わりましたら、`コミットとプッシュ`をして、`プルリクエストに修正内容が反映されているか確認`しましょう。
②既に頂いている挙動確認の画像や動画が修正範囲だった場合、最新の画像や動画の提出をお願い致します。
③googleフォームの送信をお願いいたします。
```


ご不明点や疑問点ございましたら、チャット・コネクトメンターにご相談ください。
どうぞよろしくおねがいします。

masukazuno commented:
修正しました

maimai0401 commented:
頂いたレビュー依頼ですが、修正点の差分があがっていないようです。
リモートのGithubの反映が遅いだけであれば申し訳ございません。

お手数ですがコミットとプッシュのし忘れがないか確認して頂けますでしょうか。
修正点の差分を出して再度ご提出をお願いいたします。

ご不明点ございましたら、コネクトメンターにご相談ください。
お手数ですが、どうぞよろしくおねがいします。

masukazuno committed:
修正しました。

修正しました。

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -22,19 +22,19 @@ def getWeek
         today_plans.push(plan.plan) if plan.date == @todays_date + x
       end
-      days = { :month : (@todays_date + x).month, :date : (@todays_date+x).day, :plans : today_plans}
+      days = { :month => (@todays_date + x).month, :date => (@todays_date+x).day, :plans => today_plans}
```
ハッシュロケットのシンボル化が正しく行われていないようです。
修正しましょう。

maimai0401 replied:
https://github.com/masukazuno/training_curriculum_rails7/pull/7/files#r1441563308

修正が確認できませんでした。
再度修正しましょう。

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -3,7 +3,7 @@ class calendarsController < applicationController
-    @plan = plan.new
+    @plan = Plan.new
   end
 
```
issue2の修正が確認できませんでした。
再度修正しましょう。

maimai0401 commented on app/views/calendars/index.html.erb:
```diff
@@ -22,5 +22,4 @@
         <% end %>
```
まだファイル内に不要なdivタグが存在しているようです。

修正しましょう。

maimai0401 reviewed (commented):
issue1-3のレビュー依頼ありがとうございます。プルリクエスト確認させていただきました。

★前回のコメントにあった修正点が綺麗に直せましたね。素晴らしいです。
★可読性の高い綺麗なコーディングにしようという意識が伝わるコードです。素晴らしいですね。
★完成度高く実装されているかと思われます。今までの学習がバッチリ成果として出ていますね。大変素晴らしいです。

修正点をコメントさせて頂きましたので、ご確認をお願い致します。

```ruby
【次回プルリクエスト提出する際の注意点】
①修正が終わりましたら、`コミットとプッシュ`をして、`プルリクエストに修正内容が反映されているか確認`しましょう。
②googleフォームの送信をお願いいたします。
```

ご不明点や疑問点ございましたら、チャット・コネクトメンターにご相談ください！
どうぞよろしくおねがいします。

masukazuno committed:
修正しました２

修正しました２

masukazuno committed:
修正しました

修正しました

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -1,14 +1,14 @@
-    getWeek
-    @plan = Plan.new
+    get_week
+    @plan = plan.new
```
```
    @plan = plan.new
```

上記記述は修正の必要はございません。
元の記述に戻しておきましょう。

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -1,14 +1,14 @@
-class CalendarsController < ApplicationController
+class calendarsController < applicationController
```
```
class calendarsController < applicationController
```
上記記述は修正の必要はございません。
元の記述に戻しておきましょう。

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -1,14 +1,14 @@
   # 予定の保存
   def create
-    Plan.create(plan_params)
+    plan.create(plan_params)
```
```
    plan.create(plan_params)
```

上記記述は修正の必要はございません。
元の記述に戻しておきましょう

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -18,23 +18,23 @@ def plan_params
 
     # Dateオブジェクトは、日付を保持しています。下記のように`.today.day`とすると、今日の日付を取得できます。
-    @todays_date = Date.today
+    @todays_date = date.today
```
@todays_date = date.today


上記記述は修正の必要はございません。
元の記述に戻しておきましょう。

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -18,23 +18,23 @@ def plan_params
     @week_days = []
 
-    plans = Plan.where(date: @todays_date..@todays_date + 6)
+    plans = plan.where(date: @todays_date..@todays_date + 6)
```
plans = plan.where(date: @todays_date..@todays_date + 6)


上記記述は修正の必要はございません。
元の記述に戻しておきましょう。

maimai0401 reviewed (commented):
レビュー依頼ありがとうございます。プルリクエスト確認させていただきました。

修正点をコメントさせて頂きましたので、ご確認をお願い致します。

【次回プルリクエスト提出する際の注意点】
```ruby
①修正が終わりましたら、`コミットとプッシュ`をして、`プルリクエストに修正内容が反映されているか確認`しましょう。
②既に頂いている挙動確認の画像や動画が修正範囲だった場合、最新の画像や動画の提出をお願い致します。
③googleフォームの送信をお願いいたします。
```


ご不明点や疑問点ございましたら、チャット・コネクトメンターにご相談ください。
どうぞよろしくおねがいします。

masukazuno committed:
修正しました3

修正しました3

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -3,7 +3,7 @@ class CalendarsController < ApplicationController
   # １週間のカレンダーと予定が表示されるページ
   def index
     getWeek
```
一部issue2の修正が確認できませんでした。
再度修正しましょう。

maimai0401 reviewed (commented):
レビュー依頼ありがとうございます。プルリクエスト確認させていただきました。

修正点をコメントさせて頂きましたので、ご確認をお願い致します。

【次回プルリクエスト提出する際の注意点】
```ruby
①修正が終わりましたら、`コミットとプッシュ`をして、`プルリクエストに修正内容が反映されているか確認`しましょう。
②既に頂いている挙動確認の画像や動画が修正範囲だった場合、最新の画像や動画の提出をお願い致します。
③googleフォームの送信をお願いいたします。
```


ご不明点や疑問点ございましたら、チャット・コネクトメンターにご相談ください。
どうぞよろしくおねがいします。

masukazuno committed:
修正しました4

修正しました4

maimai0401 commented on app/controllers/calendars_controller.rb:
```diff
@@ -34,8 +34,7 @@ def getWeek
       end
-      days = { :month => (@todays_date + x).month, :date => (@todays_date+x).day, :plans => today_plans}
-      @week_days.push(days)
+      days = { :month : (@todays_date + x).month, :date : (@todays_date+x).day, :plans : today_plans}
```
https://github.com/masukazuno/training_curriculum_rails7/pull/7/files#r1441515661

修正が確認できませんでした。
修正しましょう。

maimai0401 replied:
days = { :month : (@todays_date + x).month, :date : (@todays_date+x).day, :plans : today_plans}

正しいシンボル型になっていないようです。
カリキュラムを確認し修正しましょう。

maimai0401 reviewed (commented):
レビュー依頼ありがとうございます。プルリクエスト確認させていただきました。

修正点をコメントさせて頂きましたので、ご確認をお願い致します。

【次回プルリクエスト提出する際の注意点】
```ruby
①修正が終わりましたら、`コミットとプッシュ`をして、`プルリクエストに修正内容が反映されているか確認`しましょう。
②既に頂いている挙動確認の画像や動画が修正範囲だった場合、最新の画像や動画の提出をお願い致します。
③googleフォームの送信をお願いいたします。
```


ご不明点や疑問点ございましたら、チャット・コネクトメンターにご相談ください。
どうぞよろしくおねがいします。

masukazuno committed:
訂正しました

何度かメンターさんに質問して訂正しても
合格しないので申し訳ないですが
どの様なコードを記入して欲しいのか
直接書き込んで頂けませんか？
僕はこれ以上何も出来ません。

maimai0401 commented on app/views/calendars/index.html.erb:
```diff
@@ -22,6 +22,4 @@
-  </div>
-</div>
\ No newline at end of file
+    <% end %>
```
必要なdivタグが削除されているようです。
26行目にdivタグを追記しましょう。

maimai0401 reviewed (commented):
issue1-3のレビュー依頼ありがとうございます。プルリクエスト確認させていただきました。

修正点をコメントさせて頂きましたので、ご確認をお願い致します。

```ruby
【次回プルリクエスト提出する際の注意点】
①修正が終わりましたら、`コミットとプッシュ`をして、`プルリクエストに修正内容が反映されているか確認`しましょう。
②googleフォームの送信をお願いいたします。
```

ご不明点や疑問点ございましたら、チャット・コネクトメンターにご相談ください。
どうぞよろしくおねがいします。

masukazuno committed:
修正しました

修正しました

OhyaYouki reviewed (commented):
レビュー依頼ありがとうございます。

無事全てのissueを解決することができましたね！
LGTMとさせていただきます！実装お疲れさまでした！

こちらのブランチは忘れずにMergeいただき、ローカルのMasterにPullした後に次の実装のためのブランチを切って次の課題に取り組みましょう！応援しております！
